ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/tls/TlsConfigurationTest.scala

```diff
@@ -35,7 +36,7 @@ class TlsConfigurationTest extends AnyWordSpec with Matchers with BeforeAndAfter
     List("server.crt", "server.pem", "ca.crt", "client.crt", "client.pem").map { src =>
       new File(rlocation("ledger/test-common/test-certificates/" + src))
     }
-  }
+  }: @nowarn("msg=match may not be exhaustive")
```

Comment:
I find the value of this pattern gets diminished with this new annotation :) 

```
  private val prefix = "ledger/test-common/test-certificates"
  private def path(fileName: String) = new File(rlocation(s"$prefix/$fileName"))
  private val certChainFilePath = path("server.crt")
  private val privateKeyFilePath = path("server.pem")
  private val trustCertCollectionFilePath = path("ca.crt")
  private val clientCertChainFilePath = path("client.crt")
  private val clientPrivateKeyFilePath = path("client.pem")
```

Reply:
I have gotten rid of all the `nowarn` constructs now.